### DISCUSSION TITLE
[Crunchyroll] Subtitles...

### DISCUSSION THREAD

**Discussion by napgiver** [anidl/multi-downloader-nx#729] [Q&A]
New to using this software. And, I'm using the GUI release. Two things though...

1) Is there a way to _not_ get the Closed Caption subtitles?  I don't need or want them.

2) Is there anyway to change _ScaledBorderAndShadow: yes_, to _no?_ I tried adding the following to the _cli-default.yml_ file...

```
# Scaled Border & Shadow
ScaledBorderAndShadow: no
```

But, that didn't help. If there's no way at the moment to change this, then I guess this part is a feature request 🤔 As of right now, I have to open the sub file in a text editor and either delete the line, or, change the _yes_ to a _no._

**Comment by Jaynator495** [comment]
Uhhh, I didn't think anyone would ever want to disable that, but yes, there is a way to disable that if you're comfortable editing a code file and compiling yourself. Depending on the service the requirement changes, but for crunchy just comment out line 2150. I guess I can add a flag, but may I ask why you want to disable that?

As for the captions, soon, I'm in the middle of a pretty large refactor which should allow for that

**Comment by napgiver** [comment]
Compiling is not something I've ever been able to accomplish 😟 TBH I'm surprised people want the Border & Shadows to be scaled up like that. I tend to open up all the sub file I get in Aegisub to make sure timing, fades and whatever else are all correct. If it were possible to undo it in Aegisub, it wouldn't even be a problem. But, the only way to change it is in a text editor.

Would having an option to have something in the _cli-default_ file be possible?
